examples/playground/src/App.tsx

```diff
@@ -1,47 +1,175 @@
 import { useState } from 'react';
-import type { JSONSchema7 } from 'json-schema';
 import type { UiSchema } from '@rjsf/core';
-import { VernaContextProvider } from '@openfun/verna';
+import VernaProvider, { Section, VernaJSONSchemaType } from '@openfun/verna';
 import FormWrapper from './FormWrapper';
 import TextWidget from './widgets/TextWidget';
 import PasswordWidget from './widgets/PasswordWidget';
 import QuizWidget from './widgets/QuizWidget';
 import './styles/verna.scss';
+import NumberWidget from './widgets/NumberWidget';
+import SelectWidget from './widgets/SelectWidget';
 
 function App() {
-  const schemaDefault: JSONSchema7 = {
+  const schemaDefault: VernaJSONSchemaType = {
     description: 'Desc registration form',
-    properties: {},
+    properties: {
+      section: {
+        properties: {
+          onlyNumbersString: { pattern: '^\\d*$', type: 'string' },
+          select: {
+            description: 'description',
+            enum: ['Item 1', 'Item 2', 'Item 3'],
+            title: 'Select items',
+            type: 'string',
+          },
+        },
+        type: 'object',
+      },
+    },
     title: 'A registration form',
     type: 'object',
   };
 
   const formData = {};
 
   const widgets = {
+    SelectWidget: SelectWidget,
```

Comment:
```suggestion
    selectWidget: SelectWidget,
```

Reply:
Just for information, the default widgets of the lib are named with a capital letter, so SelectWidget would override the default SelectWidget

Reply:
Oh ok! And does other widgets start with a lower case letter ? Because if not, maybe the right solution is to use an uppercase for all widgets.

```
const widgets = {
    numberWidget: NumberWidget,
    passwordWidget: PasswordWidget,
    quizWidget: QuizWidget,
    selectWidget: SelectWidget,
    textWidget: TextWidget,
  };
```